CHANGELOG

```diff
@@ -1,6 +1,10 @@
-# 3.11.1 (2024-XX-XX)
+# 3.12.0 (2024-XX-XX)
 
- * n/a
+ * Bump minimum PHP version to 8.0
```

Comment:
Dropping PHP < 8.0.
Needed for the YieldReady attribute merged a while ago, and here for many features only part of PHP 8+.
The PHP 7.x versions are not maintained anymore anyway.

Reply:
should we go for PHP 8.1 to match the min version of the Symfony 6.4 LTS ?

Reply:
Nothing is interesting in PHP 8.1 for Twig. I'd like to bump just the minimum needed.